src/operator/account/MigrateOrgsOverlay.tsx

```diff
@@ -0,0 +1,105 @@
+import React, {FC, useContext} from 'react'
+import {
+  Alert,
+  ButtonBase,
+  ButtonShape,
+  ComponentColor,
+  ComponentStatus,
+  Gradients,
+  IconFont,
+  Overlay,
+  RemoteDataState,
+} from '@influxdata/clockface'
+import {AccountContext} from 'src/operator/context/account'
+import {OperatorAccount} from 'src/client/unityRoutes'
+
+interface Props {
+  toAccount: OperatorAccount
+}
+
+const MigrateOrgsOverlay: FC<Props> = ({toAccount}) => {
+  const {
+    account,
+    organizations,
+    migrateStatus,
+    handleMigrateOrgs,
+    setMigrateOverlayVisible,
+    migrateOverlayVisible,
+  } = useContext(AccountContext)
+
+  const migrateAccountOrgs = () => {
+    if (account?.type !== 'cancelled') {
```

Comment:
Is this just an extra guard in case there's a race condition, like the account being canceled while the overlay is being opened - since we only render an empty fragment instead of the `MigrateOrgsOverlay` if the account is canceled?

Since this `try / catch` block is only hit if the account type isn't canceled, might still want a notification here so that support doesn't presume the migration works if they hit the migration button and nothing happens.